### DISCUSSION TITLE
In need of receipt for my purchase

### DISCUSSION THREAD

**Discussion by Stargazer28** [aclap-dev/video-downloadhelper#1301] [License & Purchase requests]
### What's your issue?

Hi all/team,

I would like to find out how to request a receipt for my recent "Video DownloadHelper" purchase?
I received the license key after purchase but I never received a confirmation email (otherwise would have reached out there).

Is there a way to be contacted by the team directly and to issue the receipt for my purchase?

Thanks in advance!

Best,
Stargazer

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi, please send us an email at licenses@downloadhelper.net
